### PR TITLE
[GUI] Feature/Bug CoinControl Update on Open

### PR DESCRIPTION
**Pull request by Liquid369**
Improvement on #1189 
Tested working

### CODE REVIEW

File: src/qt/coincontroldialog.h [hunk 1]
```diff
@@ -49,6 +49,7 @@ class CoinControlDialog : public QDialog
     void setModel(WalletModel* model);
     void updateDialogLabels();
     void updateView();
+    void updateCoinControlLabels()
```
Comment (random-zebra) [inline src/qt/coincontroldialog.h]: missing semicolon

**Comment by Liquid369** [comment]
Fixed/Squashed

File: src/qt/coincontroldialog.cpp [hunk 2]
```diff
@@ -928,3 +928,11 @@ void CoinControlDialog::updateView()
     sortView(sortColumn, sortOrder);
     ui->treeWidget->setEnabled(true);
 }
+
+void CoinControlDialog::updateCoinControlLabels()
```
Comment (furszy) [inline src/qt/coincontroldialog.cpp]: Please change the name to `updateDialog()` or `refreshDialog()`. The method is updating/refreshing the entire component, items recreation included, and not only the labels.

If you need a hand with git let me know and can do it for you.

Comment (Liquid369) [reply src/qt/coincontroldialog.cpp]: Renamed refreshDialog();
Commited/squashed to 1 commit.
Anything else?

**Pull request merged**